### ISSUE TITLE
Offline when wire:poll="#refresh"

### ISSUE THREAD

**Issue by erikscheepers** [livewire/livewire#188]
**Describe the bug**
When you use `wire:poll="$refresh"` to refresh a component and the internet connection goes offline or you server crashes, the component does not update anymore. You still see the last status of the component. Would be nice you can catch this and trigger an 'Offline' error message. Also when you do `wire:click` when your internet is down, you also don't get a response.

**To Reproduce**
1. 
```html
<div wire:poll="$refresh">
    {{ $changing_string }}
</div>
```
2. Turn off your internet connection
3. The `$changing_string` does not update anymore and you don't see an error message

**Expected behavior**
Would be nice to be able to catch an incorrect response so you can show an error message.

**Comment by oliverbj** [comment]
This could be really useful (at least for my specific case). Maybe a generic network message - or even as  a standard component we could extend?

**Comment by erikscheepers** [comment]
Issue has been fixed in version [v0.3.9](https://github.com/livewire/livewire/releases/tag/v0.3.9)
Thanks, great solution!
